lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,28 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class to feel yourself real driver
+  class Car
+    attr_accessor :color, :model, :current_speed, :sets, :year
+
+    def self.default_car
+      Car.new {}
```

Comment:
1) self is the same as Car
2) we have no error if we delete {} but if we try to make StateAndBehaviour::Car.default_car we catch an error